core/src/main/java/com/sequenceiq/cloudbreak/service/upgrade/rds/RdsUpgradeOrchestratorService.java

```diff
@@ -40,10 +47,14 @@ public class RdsUpgradeOrchestratorService {
 
     private static final String PREPARE_UPGRADE_EMBEDDED_DATABASE = "postgresql/upgrade/prepare-embedded";
 
+    private static final String GET_EXTERNAL_DB_SIZE = "postgresql/upgrade/external-db-size";
+
     private static final int DB_SPACE_MULTIPLIER = 3;
 
     private static final int KB_TO_MB = 1024;
 
+    private static final int KB_TO_BYTE = 1024;
```

Comment:
it's weird, KB_TO_MB is the same as KB_TO_BYTE? I know your variable is a good value but little bit confused here

Reply:
With the differently named variable I just wanted to indicate that when it's used it is used for kb to byte conversion. Would it make more sense to keep just one variable named in some generic way?

Reply:
displaying is done now in MB, changed the variable accordingly so the naming makes more sense now